app/services/salesman/update.rb

```diff
@@ -0,0 +1,38 @@
+module Services
+  module Salesman
+    class Update
+      def initialize(id, name, status)
```

Comment:
you must use "variable:" because then when it fails you know exactly what you expect to be passed as a parameter.

ex: def initialize(id:, update_params)